<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerWidget</class>
 <widget class="QWidget" name="LoggerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Log file path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLogFilePath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbOpenExplorerToLog">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="tvLogView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
